web/multinode/src/app/components/modals/AddNewNode.vue

```diff
@@ -117,8 +117,9 @@ export default class AddNewNode extends Vue {
         try {
             await this.$store.dispatch('nodes/add', this.nodeToAdd);
             this.notify.success({ message: 'Node Added Successfully' });
-        } catch (error) {
+        } catch (error: any) {
```

Comment:
instead of doing it explicitly for every catch block you can go to tsconfig.ts file and add such a property

```
"useUnknownInCatchVariables": false,
```

in that case all those vars would be of type `any` by default

P.S. when you do that please revert the change for .eslintrc.js file

Reply:
Done.